lib/project_types/script/messages/messages.rb

```diff
@@ -130,6 +130,8 @@ module Messages
             \s\sOptions:
               \s\s{{command:--API_key=<API_key>}} API key. Must be a valid API key, otherwise store access fails.
               \s\s{{command:--shop_domain=<my_store.myshopify.com>}} Test store URL. Must be an existing test store.
+              \s\s{{command:--config_props=name1:value1, name2:value2}} Optional. Define the configuration of your script by passing individual name and value pairs. If used with --config_file, then matching values in --config_props will override those set in the file.
```

Comment:
We should probably put quotes around the props here as per your comment https://github.com/Shopify/shopify-app-cli/pull/727#issuecomment-654486870

```suggestion
              \s\s{{command:--config_props='name1:value1, name2:value2'}} Optional. Define the configuration of your script by passing individual name and value pairs. If used with --config_file, then matching values in --config_props will override those set in the file.
```